src/components/Login.js

```diff
@@ -27,25 +51,25 @@ function LoginScreen() {
   }
 
   return (
-    <form className="loginScreen">
-      <input
+    <Form className="loginScreen">
```

Comment:
Styled components automatically generate classes, so you can remove yours :)

Reply:
Thx :) I deleted all ClassNames :)